File: log.py
def log(tag, text):
	# Info tag
	if(tag == 'i'):
		print("[INFO] " + text)
	# Warning tag
	elif(tag == 'w'):
		print('[WARN] ' + text)
	# Error tag
	elif(tag == 'e'):
		print("[ERROR] " + text)
	# Success tag
	elif(tag == 's'):
		print("[SUCCESS] " + text)

File: utils/log.js
var chalk = require('chalk');

module.exports = function(type, msg) {

    switch(type) {
        case 'e': {
            return console.error(chalk.red(`[error]: ${msg}`));
        }
        case 's': {
            return console.log(chalk.green(`[success]: ${msg}`));
        }
        case 'w': {
            return console.warn(chalk.yellow(`[warning]: ${msg}`));
        }
        case 'i': {
            return console.info(chalk.dim(`[info]: ${msg}`));
        }
        case 'd': {
            return console.debug(chalk.blue(`[debug]: ${msg}`));
        }
        default: {
            return console.log(chalk.reset(msg));
        }
    }
}